### PR TITLE
UPDATED: Summary widget

### PR DESCRIPTION
**Pull request by Simperfy**
1. Added feature to enable summary widget to consume API.
2. Decoupled Sub & Main summary widget text count.

### CODE REVIEW

File: lib/util/constants.dart [hunk 1]
```diff
@@ -2,10 +2,15 @@ import 'package:flutter/material.dart';
...
 
 final Color totalCasesBgColor = Color(0xff56CCF2).withOpacity(0.85);
 final Color activeCasesBgColor = Color(0xffF2C94C);
 final Color recoveredBgColor = Color(0xff6FCF97).withOpacity(0.65);
-final Color diedBgColor = Color(0xffBDBDBD).withOpacity(0.80);
\ No newline at end of file
+final Color diedBgColor = Color(0xffBDBDBD).withOpacity(0.80);
+
+final double summarySubFontSize = 20;
+final double summaryMainFontSize = 52;
```
Comment (DauntlessDev) [inline lib/util/constants.dart]: Great practice! It is really good to put literal values in variables and put them in a separate file.

File: lib/widgets/summary_widget/main/main_text_count.dart [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/cupertino.dart';
+
+class TextCountWidget extends StatelessWidget {
+  final _count;
+  final double _fontSize;
```
Comment (DauntlessDev) [inline lib/widgets/summary_widget/main/main_text_count.dart:5]: Encapsulation 👏👏

File: lib/widgets/summary_widget/main/main_text_count.dart [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  TextCountWidget(this._count, this._fontSize);
```
Comment (DauntlessDev) [inline lib/widgets/summary_widget/main/main_text_count.dart:6]: I recommend using { } and @required in every widgets that has 2 or more parameters.

Comment (Simperfy) [reply lib/widgets/summary_widget/main/main_text_count.dart:6]: Noted

**Pull request merged**